### ISSUE TITLE
Path won't work like normal. Focus targets duty member to my right.

### ISSUE THREAD

**Issue by CAngeley** [ffxivcode/AutoDuty#94]
I tried resetting plugin data and reloading but no work. I've tried this on Skydeep Cenote, Worqor Zormor, and Aitiascope and each time it does the same thing. I've only done it in duty support, but it will focus target the npc to my right and follow them.

**Comment by adambennett55** [comment]
when pathing or on a boss?

**Comment by CAngeley** [comment]
When pathing. For example when on reaper in skydeep cenote, Once the duty circle is gone, it'll focus target Alphinaud and follow him constantly. This will go on for the entirety of the build path, acting like it is pathing when it's not. The only commands it seems to follow are AutoMove and Wait. After around 2mins it'll complete the build path for the dungeon and leave

**Comment by adambennett55** [comment]
sounds like a BMR issue, bring up the BMRAI ui and make sure follow out of combat is not checked

**Comment by CAngeley** [comment]
Now it's not focus targetting alphinaud but targeting myself which I believe is what it used to do. But now my character is not moving unless the step on the build path says automove

**Comment by hx0requiem** [comment]
Yeah, I've been trying it on Vanaspati and the path glitches out before changing to the second area (after the second boss) and it runs all the way back until it hits a wall. And I also noticed in Ktisis Hyperboreia that my character would suddenly stop doing the rotation and would only continue doing it again if you manually press a key (although that might be more of an RSR problem)

**Comment by CAngeley** [comment]
Not sure what exactly I did but I turned off all mods related to getting autoduty to work, went to bed and then next morning turned them on (and update any needed) and it all is working again like expected.
